tox.ini

```diff
@@ -97,7 +116,6 @@ commands =
     blackdoc --check --diff {[testenv]lint_folders}
     isort --check-only --diff {[testenv]lint_folders}
 
-
```

Comment:
I have the feeling every time we touch the `tox.ini` file we either add or delete lines between the sections 🤣